Source/Core/Core/IOS/STM/STM.h

```diff
@@ -55,7 +55,7 @@ class STMEventHook final : public Device
 {
 public:
   using Device::Device;
-  IPCCommandResult Close(u32 fd) override;
+  ~STMEventHook();
```

Comment:
`override`?

Reply:
Whoops, fixed. (Weird that I got no warnings from this)